components/server/src/external/database/reports.py

```diff
@@ -102,7 +102,7 @@ def _prepare_documents_for_insertion(
             document[key] = value
 
 
-def latest_report(database: Database, data_model, report_uuid: str) -> Report | None:
+def latest_report(database: Database, data_model, report_uuid: str):
```

Comment:
Why did you remove the return type?

Reply:
Because otherwise mypy starts complaining about bad typing in all endpoints where this function is used without covering the case that the report is None (report_uuid doesnt exist, these are about 15 different endpoints for reports and notifications).

The solution is of course to implement a 404 error because the specified report has not been found. However, I think this should fall outside of the scope of this story. Instead I made story #3717 to solve this.

Reply:
Ah, clear.